src/ui/Startup.cs

```diff
@@ -47,9 +49,23 @@ public void ConfigureServices(IServiceCollection services)
                 loggingBuilder.AddSerilog(dispose: true));
 
             var sharedAssembly = typeof(CourseDetailsViewComponent).GetTypeInfo().Assembly;
+
+
             services.AddMvc(options =>
                 options.Filters.Add(typeof(McExceptionFilter))
-            ).AddApplicationPart(sharedAssembly);
+            ).ConfigureApplicationPartManager(apm =>
+            {
+                var apiClientAssembly = typeof(GovUk.Education.ManageCourses.ApiClient.ManageCoursesApiClient).GetTypeInfo().Assembly.GetName().Name;
+            var api = typeof(GovUk.Education.ManageCourses.Api.Startup).GetTypeInfo().Assembly.GetName().Name;
+                var dependentLibraries = apm.ApplicationParts.Where(x  => x.Name == api || x.Name == apiClientAssembly).ToList();
+
+                foreach (var item in dependentLibraries)
+                {
+                    apm.ApplicationParts.Remove(item);
```

Comment:
perhaps a comment explaining why this is necessary

Reply:
When you comment it out you know why.

Reply:
superfluous

Application Part Manager removes an Application Part that has the assembly with stated name